src/translations/en.ts

```diff
@@ -493,14 +493,17 @@ export const en = {
     'success.order.canceled': 'Order was cancelled',
     'success.order.canceled.all': 'All orders were cancelled',
     'success.order.cancelling.all': 'All orders are being canceled',
-    'success.order.created': 'Order was created',
     'success.phone.verification.send': 'Verification code was sent to your phone',
     'success.phone.confirmed': 'Your phone was confirmed',
     'success.phone.confirmation.message': 'Success!',
     'success.message.sent': 'Message was sent',
     'success.email.confirmed': 'Your email address has been successfully confirmed',
+    'success.order.created': 'Order was created',
+    'success.order.done': 'Order successfully completed',
 
     // error messages
+    'error.order.rejected': 'Order rejected',
```

Comment:
Please add the following strings:
- "error.invalid_request"
- "error.bad_request"
- "error.request_entity_too_large"